crates/esthri-cli/Cargo.toml

```diff
@@ -26,7 +26,7 @@ name = "esthri_server"
 path = "src/http_server.rs"
 
 [dependencies]
-esthri = { path = "../esthri", default-features = false }
+esthri = { version = "8.4.0", path = "../esthri", default-features = false }
```

Comment:
Last version we published to GitHub was 8.4.0-- is that not on crates.io?

Reply:
Meant to say 8.5.0 here, updated comment.

Reply:
We have created a local 8.5.0 release but have only pushed up to 8.4.0 to crates.io. When you run the `cargo release` command it will replace the version specified with the new one you are publishing. This is useful because we are effectively publishing 2 crates at the same time. So it will replace the strings in the cargo.toml, publish esthri 8.5.1, then publish esthri-cli (now using the 8.5.1 version of esthri instead of the local path used for workspaces).

Reply:
👍